### PR TITLE
make certification script's interface more clear

### PR DESCRIPTION
**Pull request by scottbartell**
Also adds nifty usage information:

image

### CODE REVIEW

File: exe/vantiv-certify-app [hunk 1]
```diff
@@ -2,22 +2,45 @@
...
+  end
+
+  opts.on("--filter_by [TEST_NAME]", "Filter which certifications to run") do |filter|
+    options[:filter_by] = filter
+  end
+end.parse!
+
+required_arguments = %i[acceptor_id license_id paypage_id]
+required_arguments.each do |required_argument|
+  raise OptionParser::MissingArgument if options[required_argument].nil?
```
Comment (joshuaballoch) [inline exe/vantiv-certify-app]: what about 1. finding all of the missing arguments, and 2. listing the missing arguments when raising this error

Comment (ChrisVoxland) [inline exe/vantiv-certify-app]: getting `undefined method `MissingArgument' for OptionParser:Class (NoMethodError)` when i try to run this with no args. doesn't seem like the error we want to be throwing.

File: README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ Vantiv's DevHub requires merchants to certify their applications for usage with
 To certify your application, run the following script:
 
 ```
-$ LICENSE_ID=sub-your-license-id-in-here ACCEPTOR_ID=sub-your-acceptor-id-in-here PAYPAGE_ID=your-paypage-id vantiv-certify-app
+$ vantiv-certify-app -l sub-your-license-id-in-here -a sub-your-acceptor-id-in-here -p your-paypage-id
```
Comment (ChrisVoxland) [inline README.md]: shouldn't this be `bundle exec vantiv-certify-app`?

**Comment by scottbartell** [comment]
When options are missing:

image

**Comment by ChrisVoxland** [comment]
👍 once readme is updated

**Pull request merged**